ci/bazel-scripts/diff.sh

```diff
@@ -13,7 +13,11 @@ set -euo pipefail
 set -x
 cd "$(git rev-parse --show-toplevel)"
 
-MERGE_BASE="${MERGE_BASE_SHA:-HEAD}"
+# since default branch is not the same in ic-private, we can't set it to master, throw error if missing
+git fetch origin $DEFAULT_BRANCH
```

Comment:
This would cause quite some downloads since [we don't fetch all history when we checkout the repo](https://sourcegraph.com/github.com/dfinity/ic/-/blob/.github/workflows/ci-main.yml?L46).

Reply:
But it's only pulling master... Or should I specify a couple commits like we have in the checkout step?

Reply:
Using `--depth 512` should be safe enough.